packages/expo-payments-stripe/ios/EXPaymentsStripe/EXTPSStripeManager.m

```diff
@@ -857,11 +857,10 @@ -(void)callGraphqlApiWithQuery:(NSString *)query showAlertOnError:(bool)displayA
         if (error || json[@"errors"]) {
           // exact errors will be captured by sentry
           NSString *defaultErrorMessage = displayAlert ? @"createCart service failed" : @"Invalid Shipping Address" ;
-          NSDictionary *firstError = [json[@"errors"] firstObject];
-          NSString *errorMessage = firstError ? [firstError objectForKey:@"message"] : defaultErrorMessage;
+          NSDictionary *firstError = json[@"errors"];
           if (displayAlert){
             //extracting the message from the api
-            NSException *exceptionFromService = [NSException exceptionWithName:@"service failed" reason:errorMessage userInfo:@{}];
+            NSException *exceptionFromService = [NSException exceptionWithName:@"service failed" reason:firstError userInfo:@{}];
```

Comment:
@neiltheyes  - just for my understanding json[@"errors"] is a list and not a dictionary . why aren't we picking the first object here ?